k8s/helm-chart/dynatrace-gcp-function/values.yaml

```diff
@@ -33,7 +33,9 @@ dynatraceUrl: ""
 
 # subscription id of log sink pubsub subscription
 logsSubscriptionId: ""
-#dynatraceLogIngestUrl ActiveGate endpoint used to ingest logs to Dynatrace, for example: https://environemnt-active-gate-url:9999/e/environment-id"
+#dynatraceLogIngestUrl ActiveGate endpoint used to ingest logs to Dynatrace:
+#  - for Public ActiveGate: https://<your_environment_ID>.live.dynatrace.com
```

Comment:
'Public ActiveGate' is our internal term.
rephrase to something like:
"- for direct cluster ingest"

Reply:
You're right! I only learned about it today. We have a discussion ongoing with Teo and Michał F. on how to call it so that it's also consistent with our official docs